contracts/modules/cover/Cover.sol

```diff
@@ -529,6 +529,11 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
     return ICoverNFT(coverNFT).ownerOf(coverId);
   }
 
+  function stakingPoolTransferTokens(uint stakingPoolId, address from, address to, uint[] calldata tokenIds) external {
```

Comment:
I think this is missing an onlyInternal modifier?

Reply:
ooof. good find